API.md

```diff
@@ -265,6 +265,7 @@
             * [.hasTraits()](#module_@asyncapi/parser+Operation+hasTraits) ⇒ <code>boolean</code>
             * [.messages()](#module_@asyncapi/parser+Operation+messages) ⇒ <code>Array.&lt;Message&gt;</code>
             * [.message()](#module_@asyncapi/parser+Operation+message) ⇒ <code>Message</code>
+            * [.OperationSecurityRequirement](#module_@asyncapi/parser+OperationSecurityRequirement) ⇐ <code>Base</code>
```

Comment:
🤔 where is this defined?

Reply:
This doc should be generated automatically iirc. 
Indeed `OperationSecurityRequirement`is not the method you declared, but `Security`.

Reply:
Thanks @fmvilas and @smoya 
If its generated automatically, would you suggest reverting my changes?

Reply:
Would you mind trying `npm run docs`? This command should build the right API doc.

Reply:
The whole PR is ok, just delete the above and I can accept :)

Reply:
@smoya @magicmatatjahu I deleted my changes to the API doc and regenerated it using the above command.